--- snortalog/snortalog.pl.orig	2005-12-01 13:46:21.000000000 +0100
+++ snortalog/snortalog.pl	2006-07-21 08:48:11.084440000 +0200
@@ -5,14 +5,15 @@
 # Revision: 2.4.0
 # 
 
+use lib '/usr/share/snortalog';
 #
 # Main variables
 #
-$domains_file = "conf/domains"; $DOMAINS = 1;			# Path to find Domain file
-$rules_file = "conf/rules"; $RULES = 1;				# Path to find Rules file
-$hw_file = "conf/hw"; $HW = 1;					# Path to find Hardware file
-$lang_file ="conf/lang"; $LANG = 1;				# Path to find Language file
-$picts_dir ="picts"; $PICTS = 1;				# Path to find pictures HTML files 
+$domains_file = "/usr/share/snortalog/conf/domains"; $DOMAINS = 1;			# Path to find Domain file
+$rules_file = "/usr/share/snortalog/conf/rules"; $RULES = 1;				# Path to find Rules file
+$hw_file = "/usr/share/snortalog/conf/hw"; $HW = 1;					# Path to find Hardware file
+$lang_file ="/usr/share/snortalog/conf/lang"; $LANG = 1;				# Path to find Language file
+$picts_dir ="/usr/share/snortalog/picts"; $PICTS = 1;				# Path to find pictures HTML files 
 #$html_directory = "/tmp/";					# Default output directorys (HTML output exclusively)
 $dbm_directory = "/tmp/";					# Default output directorys (HTML output exclusively)
 $tmpout_file = "/tmp/.snortalog.tmp";				# Default tempory file (GUI exclusively)
@@ -1877,7 +1878,7 @@
 	print( '=' x $linelength, "\n");
 	print(" " x 4, "### ". $n ." of ". keys(%s14) ." ###\n");
 	print("    %    No      Actions\n");
-} elsif ($report eq "rules") {
+} elsif ($report eq "/usr/share/snortalog/rules") {
 	$linelength = 13 + $nb_len + 4 + 6;
 	print( '=' x $linelength, "\n");
 	print(" " x 4, "### ". $n ." of ". keys(%s21) ." ###\n");
